app/assets/stylesheets/dashboard.css

```diff
@@ -216,6 +216,20 @@
   margin-right: 5px;
 }
 
+/* Issue #5031 (daddyjab):
+   Revised to add border of
+   offsetting color to Twitter buttons */
+.dashboard .social-media-btn #follow-button {
+  border: 1px solid #00aced;
+}
+
+/* Issue #5031 (daddyjab):
+  Revised to add border of
+  offsetting color to Facebook buttons */
+.dashboard .social-media-btn #facebook button {
```

Comment:
Hey @daddyjab, please remove the comments.

Reply:
Hi, @gauravano
Yes, I will add a "before" screenshot, and I will remove the comments from the CSS file changes.

Regarding the feature branch, I was not able to push to a feature branch on the plots2 main repo due to permissions.  But maybe you're saying I should create a feature branch on my _forked_ repo, which I'll do for these next changes- thanks!